src/models/Enums.ts

```diff
@@ -4,6 +4,8 @@ export enum ERRORS {
     EMPTY_PAYLOAD = "with empty payload.",
     LAMBDA_INVOCATION_BAD_DATA = "Lambda invocation returned bad data:",
     RETRO_ERROR_OR_CVS_UPDATED = "Not eligible for certificate generation.",
+    SECRET_ENV_VAR_NOT_EXIST = "SECRET_NAME environment variable does not exist.",
+    SECRET_FILE_NOT_EXIST = "The secret file does not exist.",
```

Comment:
May as well use these instead of the strings.

Reply:
done